ddl/index.go

```diff
@@ -751,6 +751,14 @@ func (d *ddl) backfillBatchTasks(startTime time.Time, startHandle int64, reorgIn
 	// nextHandle will be updated periodically in runReorgJob, so no need to update it here.
 	d.reorgCtx.setNextHandle(nextHandle)
 	metrics.BatchAddIdxHistogram.Observe(elapsedTime)
+  
+  metrics.JobsGauge.WithLabelValues(reorgInfo.Type.String()).Inc()
+	timeStart := time.Now()
+	defer func() {
+		metrics.JobsGauge.WithLabelValues(reorgInfo.Type.String()).Dec()
+		metrics.BatchAddIdxHistogram.WithLabelValues(reorgInfo.Type.String(), metrics.RetLabel(err)).Observe(time.Since(timeStart).Seconds())
```

Comment:
So the observed time is always zero, right?

Reply:
Observed always no data points.